lib/livebook.ex

```diff
@@ -58,6 +58,10 @@ defmodule Livebook do
       config :livebook, :data_path, data_path
     end
 
+    if Livebook.Config.force_ssl!("LIVEBOOK_FORCE_SSL") do
+      Keyword.new() |> Plug.SSL.init()
```

Comment:
You need to store this in config, something like `config :livebook, :force_ssl, true | false`. And then, you need a plug in the endpoint.ex like this:

```elixir
plug :force_ssl

@plug_ssl Plug.SSL.init([])
def force_ssl(conn, _opts) do
  if Application.get_env(:livebook, :force_ssl, false) do
    conn
  else
    Plug.SSL.call(conn, @plug_ssl)
  end
end
```

There are probably other details, but this is the minimum required to get started!

Reply:
I actually think we should call the option `LIVEBOOK_FORCE_SSL_HOST` and it should be set to a binary which is the host we redirect to!

Reply:
> I actually think we should call the option `LIVEBOOK_FORCE_SSL_HOST` and it should be set to a binary which is the host we redirect to!


OK, I'll create new PR about this soon.
I truly appreciate your comments :)